tests/mail/tests.py

```diff
@@ -1469,12 +1469,25 @@ class SMTPBackendStoppedServerTest(SMTPBackendTestsBase):
     ("RuntimeError: threads can only be started once").
     """
 
-    def test_server_stopped(self):
+    def setUp(self):
+        super(SMTPBackendStoppedServerTest, self).setUp()
+        self.backend = smtp.EmailBackend(username='', password='')
+
+    def test_1_server_stopped(self):
```

Comment:
Number is to ensure that this test which closes the server is ran first, got a better suggestion?